src/utils.jl

```diff
@@ -303,10 +303,10 @@ function convert_to_polar(theta, r, r_extrema = ignorenan_extrema(r))
     x, y
 end
 
-function fakedata(sz...)
+function fakedata(sz...; rng=StableRNG(PLOTS_SEED))
```

Comment:
```suggestion
function fakedata(sz...; rng=Random.GLOBAL_RNG(PLOTS_SEED))
```
What I wanted to say is that we can keep using the unstable RNG here, and switch to the StableRNG in the tests